### DISCUSSION TITLE
skynet可以共享userdata吗？

### DISCUSSION THREAD

**Discussion by DJKarlHarris** [cloudwu/skynet#1775] [General]
想复用proto加载的文件数据，而proto是一个userdata的数据类型

**Comment by cloudwu** [comment]
可以。看 https://github.com/cloudwu/skynet/blob/master/lualib/sproto.lua#L26

**Comment by DJKarlHarris** [comment]
云大，我不想用sproto，单纯的想复用一个userdata，请问可以做到吗

**Reply by sniper00** [reply]
Protobuf可以看看这个，https://github.com/sniper00/MoonDemo/issues/1
